pkg/diagnose/deployments.go

```diff
@@ -214,4 +222,8 @@ func checkPodsStatus(k8sClient kubernetes.Interface, namespace string, status re
 			}
 		}
 	}
+
+	if numOfActiveGws != 1 {
+		status.Warning("Number of GW pods labeled as active is not 1, (numOfActiveGws is %d)", numOfActiveGws)
```

Comment:
```suggestion
		status.Warning("Expected one Gateway pod to be labeled as active. Found %d", numOfActiveGws)
```

I think it may also be useful to identify which gateways were labeled as active. Perhaps the pod's node  name would be most useful to report. We can also use the `gateway.submariner.io/node` label. So instead of `numOfActiveGws`, we could define an `activeGwNodeNames` array and check the len here and report `status.Warning("... Found %d on nodes %v", len(activeGwNameNames), activeGwNameNames)`.